toDoList/src/index.js

```diff
@@ -1,72 +1,92 @@
+    tasks[taskIndex].completed = true;
+    taskItem.classList.add('completed');
+    checkbox.setAttribute('checked', '');
+  });
+  const unselectedCheckboxes = todoList.querySelectorAll('input[type="checkbox"]:not(:checked)');
+  unselectedCheckboxes.forEach((checkbox) => {
+    const taskItem = checkbox.parentNode;
+    const taskIndex = Array.from(todoList.children).indexOf(taskItem);
+    tasks[taskIndex].completed = false;
+    taskItem.classList.remove('completed');
+    checkbox.removeAttribute('checked');
+  });
+};
 
-  tasks.forEach((task) => {
-    const listItem = document.createElement('label');
-    const listInput = document.createElement('input');
-    const x = document.createElement('BR');
+// Function to render the tasks in the todoList
+const renderTasks = () => {
+  // Clear the todoList
+  todoList.innerHTML = '';
+
+  // Render each task
+  tasks.forEach((task, index) => {
+    const taskItem = document.createElement('li');
     const hr = document.createElement('HR');
-    listInput.type = 'checkbox';
-    listItem.classList.add('listItem');
-    listItem.textContent = task.description;
-    if (task.completed) {
-      listItem.classList.add('completed');
-    }
-    items.appendChild(listInput);
-    items.appendChild(listItem);
-    items.appendChild(x);
-    items.appendChild(hr);
-    todoList.appendChild(items);
+    const taskCheckbox = document.createElement('input');
+    taskCheckbox.type = 'checkbox';
+    taskCheckbox.checked = task.completed;
+    taskCheckbox.addEventListener('change', () => toggleCompleted(index));
+    const taskDescription = document.createTextNode(task.description);
+    taskItem.appendChild(taskCheckbox);
+    taskItem.appendChild(taskDescription);
+    taskItem.appendChild(hr);
+    todoList.appendChild(taskItem);
   });
 
-  todoList.appendChild(removeTask);
-}
+  // Update the indexes of the tasks
+  tasks.forEach((task, index) => {
+    task.index = index;
+  });
+
+  // Save the tasks to local storage
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// Function to remove all completed tasks
+const removeCompletedTasks = () => {
+  tasks = tasks.filter((task) => !task.completed);
+  renderTasks();
+};
 
-// Call the showTasks function on page load
-window.addEventListener('load', showTasks);
+// Add event listener to the addForm for submitting new tasks
+addForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const newTaskInput = document.querySelector('#new-task');
+  const newTaskDescription = newTaskInput.value.trim();
+  if (newTaskDescription !== '') {
+    addTask(newTaskDescription);
+    newTaskInput.value = '';
+  }
+});
+
+// Add event listener to the removeCompletedBtn to remove all completed tasks
+removeCompletedBtn.addEventListener('click', removeCompletedTasks);
+
+// Load saved tasks from local storage
+const savedTasks = JSON.parse(localStorage.getItem('tasks'));
+if (savedTasks) {
+  tasks = savedTasks;
+  renderTasks();
```

Comment:
- [ ] As instructed on the project requirement, there should be a function for editing tasks, and this should also be included in the new file just as I pointed out in the review on line `8`. Kindly the functionality to edit 👍 

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/222012465-a91f0087-ccc3-407a-90b4-0a81eb422e23.png)